src/test/java/uk/ac/ebi/ena/readtools/webin/cli/rawreads/refs/CramReferenceInfoTest.java

```diff
@@ -7,13 +7,15 @@
 import java.util.Map;
 
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 import uk.ac.ebi.ena.readtools.webin.cli.rawreads.BamScannerTest;
 
 public class 
 CramReferenceInfoTest 

```

Comment:
Not sure if this class is in the correct package.

Reply:
There are a bunch of classes inside the package uk.ac.ebi.ena.readtools.webin.cli.rawreads. These were all here from before. Any suggestions on where should we move them or whether it ll be better to rename the package to something else ?

Reply:
There are two clients for the readtools project. webin-cli and sra-stages. The uk.ac.ebi.ena.readtools.webin.cli.rawreads package name suggests that it is used by webin-cli. Please check which classes are used by webin-cli and sra-stages before renaming/moving packages/classes so that we don't break these build unnecessarily. The package name uk.ac.ebi.ena.readtools.webin.cli.rawreads would be ok for classes that are indeed used (I hope only) by webin-cli. The readtools is intended to be an 'API' project used by webin-cli and sra-stages and as with any 'API' project we need to understand what is the public API that needs to be stable and what is the other stuff that can change without risking breaking the clients. E.g. Sam2Fastq will be part of the public API and once it is used directly by Eugene it should remain stable unless API changes are discussed and agreed with the client(s) .

Reply:
So I checked and found that none of the two repositories use any class from this package. I made sure of it by building them two locally with the new version of readtools. I think it might be ok to just remove the package and all of the class in there completely.

Reply:
although the classes in this packages are not being used by either of the other two repositories. they are however, being used by another class uk.ac.ebi.ena.readtools.validator.ReadsValidator inside this repo that is being used by webin-cli. so, as agreed by the team, we are going to keep the package as it is for now.